curriculum/challenges/english/08-coding-interview-prep/rosetta-code/levenshtein-distance.md

```diff
@@ -0,0 +1,81 @@
+---
+id: 5a23c84252665b21eecc7ee2
+title: Levenshtein distance
+challengeType: 5
+---
+
+## Description
+<section id='description'>
+In information theory and computer science, the <b>Levenshtein distance</b> is a <a href="https://en.wikipedia.org/wiki/string metric">metric</a> for measuring the amount of difference between two sequences (i.e. an <a href="https://en.wikipedia.org/wiki/edit distance">edit distance</a>). The Levenshtein distance between two strings is defined as the minimum number of edits needed to transform one string into the other, with the allowable edit operations being insertion, deletion, or substitution of a single character.
+Example:
+The Levenshtein distance between "<b>kitten</b>" and "<b>sitting</b>" is 3, since the following three edits change one into the other, and there isn't a way to do it with fewer than three edits:
+<b>k</b>itten   <b>s</b>itten      (substitution of 'k' with 's')</b>
+sitt<b>e</b>n   sitt<b>i</b>n      (substitution of 'e' with 'i')</b>
+sittin         sittin<b>g</b>     (insert 'g' at the end).</b>
+''The Levenshtein distance between  "<b>rosettacode</b>",  "<b>raisethysword</b>"  is  <b>8</b>.
+<i>The distance between two strings is same as that when both strings are reversed.</i>
+</section>
+
+## Instructions
+<section id='instructions'>
+Write a function that returns the Levenshtein distance between two strings given as parameters .
```

Comment:
We can remove the space before the period here:

Write a function that returns the Levenshtein distance between two strings given as parameters.